src/style.css

```diff
@@ -0,0 +1,63 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir.ttc");
+  font-style: normal;
+  font-weight: normal;
+}
+
+body {
+  box-sizing: border-box;
+}
+
+.search__inpt {
+  width: 100%;
+  outline: none;
+  font-family: Avenir, Helvetica, sans-serif;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  color: #3d4e61;
+  font-weight: 900;
+}
+
+.search__inpt::placeholder {
+  font-weight: 300;
+}
+
+.search__inpt:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search__inpt:focus {
+  background-color: transparent;
+  border-radius: 3px;
+}
+
+.bigBar {
+  margin: 20px 0;
+}
+
+.bigBar__form,
+.smallBar__form {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+}
+
+.bigBar__inpt {
+  padding: 23px 20px 23px 62px;
+  line-height: 22px;
+  font-size: 16px;
+  background-image: url("./images/Search.svg");
+  background-repeat: no-repeat;
+  background-position: 26px 25px;
+  background-size: 19px;
+}
+
+.smallBar__inpt {
+  padding: 10px 20px 11px 33px;
+  font-size: 14px;
+  line-height: 19px;
+  background-image: url("./images/Search.svg");
+  background-repeat: no-repeat;
+  background-position: 12px 14px;
+  background-size: 11px;
+}
```

Comment:
if you have two or more similar elements with portions of similar styles with different values - use one of the elements as the basic case, and override necessary styles for other cases.